getaround_utils/lib/getaround_utils/railties/lograge.rb

```diff
@@ -20,13 +20,21 @@ def append_info_to_payload(payload)
       payload[:lograge][:user_id] = current_user&.id if defined?(current_user)
       payload[:lograge][:origin] = 'lograge'
 
-      if defined?(NewRelic::Agent::Tracer)
+      if defined?(NewRelic::Agent::Tracer) && defined?(NewRelic::Agent) && defined?(NewRelic::Agent::Hostname)
         if span_id = NewRelic::Agent::Tracer.span_id
-          payload[:lograge]["span.id"] = span_id
+          payload[:lograge]['span.id'] = span_id
         end
         if trace_id = NewRelic::Agent::Tracer.trace_id
-          payload[:lograge]["trace.id"] = trace_id
+          payload[:lograge]['trace.id'] = trace_id
         end
+        payload[:lograge]['entity.type'] = 'SERVICE'
+        if entity_guid = NewRelic::Agent.config[:entity_guid]
```

Comment:
I think maybe we can dispense ourselves of the `if`, that's a lot of them and they don't really do anything meaningful.